oracledb-ballerina/client.bal

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/crypto;
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+public client class Client {
+    *sql:Client;
+    private boolean clientActive = true;
+
+    # Initialize Oracle Client.
+    #
+    # + host - Hostname of the Oracle database server to be connected
+    # + user - Name of a user of the database
+    # + password - Password for the user
+    # + database - System Identifier or the Service Name of the database
```

Comment:
```suggestion
    # + database - System Identifier or the Service Name of the database
```

Reply:
Made the first letters of words simple with Fixed with fe49de7
``` ballerina
database - System identifier or the service name of the database
```